tools/migration/internal/runner.go

```diff
@@ -36,7 +36,7 @@ type RepoWrangler interface {
 	// GetOldRepo opens and returns the old repo with read-only access
 	GetOldRepo() (*os.File, error)
 	// MakeNewRepo creates and returns the new repo dir with read/write permissions
-	MakeNewRepo() (*os.File, error)
+	MakeNewRepo() error
 	// GetOldRepoPath returns the full path of the old repo
 	GetOldRepoPath() string
```

Comment:
The name "old" might cause confusion here. After installation, the "old" repo is at an archived path, and the new migrated repo is at the old path. Maybe something like "target" or "canonical"?

Reply:
+1 to calling it tmp first and live after